src/SearchEngine.java

```diff
@@ -0,0 +1,111 @@
+import strategies.SearchStrategy;
+import strategies.StrategyALL;
+import strategies.StrategyANY;
+import strategies.StrategyNONE;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class SearchEngine {
+
+    Map<String, ArrayList<Integer>> map = new HashMap<>();
+    ArrayList<String> list = new ArrayList<>();
```

Comment:
Rethink variables names. `map` and `list` nothing tells the end-user.

Reply:
changed

Reply:
I still see it in other places. Left comments there, so closed this conversation.